src/Core/Scheduler/Providers/Fetcher.js

```diff
@@ -14,23 +12,55 @@ function checkResponse(response) {
 
 export default {
 
-    json(url) {
-        return fetch(url).then((response) => {
+    /**
+     * Little wrapper over fetch to get some json
+     *
+     * @param {string} url
+     * @param {Object} options - fetch options (passed directly to fetch)
+     *
+     * @return {Promise}
+     */
+    json(url, options = {}) {
```

Comment:
This default was a mistake in the first place, if you ask me. Let's default to same default the browsers have, aka nothing. We're not here to rewrite the web for the user in my opinion.